_includes/pbc/all/install-features/202403.0/marketplace/install-the-marketplace-merchant-feature.md

```diff
@@ -0,0 +1,1653 @@
+
+{% endinfo_block %}
+
+**src/Pyz/Zed/OauthMerchantUser/OauthMerchantUserConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\OauthMerchantUser;
+
+use Spryker\Zed\OauthMerchantUser\OauthMerchantUserConfig as SprykerOauthMerchantUserConfig;
+
+class OauthMerchantUserConfig extends SprykerOauthMerchantUserConfig
+{
+    /**
+     * Specification:
+     * - Returns a list of configurations for endpoints accessible to merchant users.
+     * - Structure example:
+     * [
+     *      '/example' => [
+     *          'isRegularExpression' => false,
+     *      ],
+     *      '/\/example\/.+/' => [
+     *          'isRegularExpression' => true,
+     *          'methods' => [
+     *              'patch',
+     *              'delete',
+     *          ],
+     *      ],
+     * ]
+     *
+     * @api
+     *
+     * @return array<string, mixed>
+     */
+    public function getAllowedForMerchantUserPaths(): array
+    {
+        return [];
+    }
+}
+```
+
+### 4) Add Zed translations
+
+Generate new translation cache for Zed:
+
+```bash
+console translator:generate-cache
+```
+
+### 5) Set up behavior
+
+Enable the following behaviors by registering the plugins:
+
+| PLUGIN                                               | DESCRIPTION                                                                                              | PREREQUISITES | NAMESPACE                                                            |
+|------------------------------------------------------|----------------------------------------------------------------------------------------------------------|---------------|----------------------------------------------------------------------|
+| MerchantProfileExpanderPlugin                        | Expands merchant with profile data.                                                                      |               | Spryker\Zed\MerchantProfile\Communication\Plugin\Merchant            |
+| MerchantProfileMerchantPostCreatePlugin              | Creates merchant profile on merchant create action.                                                      |               | Spryker\Zed\MerchantProfile\Communication\Plugin\Merchant            |
+| MerchantProfileMerchantPostUpdatePlugin              | Updates merchant profile on merchant update action.m                                                     |               | Spryker\Zed\MerchantProfile\Communication\Plugin\Merchant            |
+| MerchantProfileContactPersonFormTabExpanderPlugin    | Adds an extra tab to merchant edit and create forms for editing and creating contact person data.        |               | Spryker\Zed\MerchantProfileGui\Communication\Plugin\MerchantGui\Tabs |
```

Comment:
Missed plugins:
* MerchantProfileAclEntityConfigurationExpanderPlugin

Reply:
@abitskil, this plugin is in `install-the-marketplace-merchant-portal-core-feature`.